roles/metrics/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+foreman_metrics_url: "https://{{ ansible_fqdn | default('localhost') }}/metrics"
```

Comment:
In what cases is `ansible_fqdn` not set?

Reply:
gather_facts: false